index.bs

```diff
@@ -5220,6 +5248,14 @@ In order to perform a [=registration ceremony=], the [=[RP]=] MUST proceed as fo
 
 1. Verify that the value of <code>|C|.{{CollectedClientData/origin}}</code> matches the [=[RP]=]'s [=origin=].
 
+1. If <code>|C|.{{CollectedClientData/topOrigin}}</code> is present:
+
+    1. Verify that the [=[RP]=] expects that this credential would have been created within an iframe that is
+        not [=same-origin with its ancestors=].
+
+    1. Verify that the value of <code>|C|.{{CollectedClientData/topOrigin}}</code> matches the [=origin=] of a page
+        that the [=[RP]=] expects to be sub-framed within.
+
```

Comment:
This check should also be added to §7.2. Verifying an Authentication Assertion, no?

Reply:
`topOrigin` is not yet set for get() - @timcappalli will be doing that follow-up work in https://github.com/w3c/webauthn/issues/1842 .

Reply:
Ah, I see. Thanks!